features/prompt-registry.mdx

```diff
@@ -138,11 +138,15 @@ They are also available through the following REST API endpoints
 
 ### Metadata
 
-Custom metadata can be associated with individual prompt template versions. This allows you to set values such as provider, model, or temperature for each prompt template. It is important to note here that `model` attribute is reserved for the model parameters.
+Custom metadata can be associated with individual prompt template versions. This allows you to set values such as provider, model, temperature, or any other key-value pair for each prompt template. While the `model` attribute is reserved for the model parameters, you can add any other custom metadata as per your requirements.
```

Comment:
should be "for each prompt template version"

Reply:
"While the `model` attribute is reserved for the model parameters, you can add any other custom metadata as per your requirements."
->
"Please note that the `model` attribute is reserved for model parameters, avoid putting custom metadata here."